samples: DFU over Bluetooth SMP rework (#117)
nrfconnect/ncs-zigbee: docs/includes/zigbee_bt_dfu_activation.txt

Activating optional extensions
------------------------------

To activate the :ref:`lib_zigbee_fota`, use the :file:`prj_fota.conf` configuration file together with the ``zigbee_fota_ext_flash`` snippet.

> The snippet provides external-flash partition layouts, sysbuild MCUboot settings, and the matching MCUboot configuration (via ``zigbee_fota_mcuboot_ext_flash``).

Reviewer: ```suggestion
The snippet provides external-flash partition layouts, sysbuild MCUboot settings, and the matching MCUboot configuration (through ``zigbee_fota_mcuboot_ext_flash``).
```

For example, when building from the command line, use the following command:

.. parsed-literal::
   :class: highlight

   west build |sample dir| -b *board_target* -- -DFILE_SUFFIX=fota -DSNIPPET=zigbee_fota_ext_flash

The FOTA variant enables `MCUboot image compression`_.
The generated Zigbee update file contains a compressed MCUboot image, and MCUboot decompresses it while applying the update.

> This setting reduces the update image size, but the MCUboot must run in the overwrite-only mode, so the device cannot revert to the previous image after the update is applied.

Reviewer: ```suggestion
This setting reduces the size of the update image. 
However, it requires MCUboot to run in overwrite-only mode, which means the device cannot revert to the previous image after an update is applied.
```
